news/2024/2024-12-05-osu-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,117 @@
+[![](/wiki/shared/news/2024-12-05-osu-world-cup-2024-finals-recap/7he_osu_world_cup_2_dariacore_7imeline_banner.jpg)](https://osu.ppy.sh/beatmapsets/2284644#osu/4872751)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/366/Songs/passchooo_-_7he_osu_world_cup_2_dariacore_7imeline_ft._Azer_Jade__Rain.mp3">
+    </audio>
+</div>
+
+[passchooo](https://osu.ppy.sh/beatmaps/artists/366)'s fun medley of osu! references in [7he osu! world cup 2 dariacore 7imeline](https://osu.ppy.sh/beatmapsets/2284644#osu/4872751) served as our second original, being a truly chaotic hidden map featuring jagged flow aim and slider control, done by none other than [wafer](https://osu.ppy.sh/users/9416836).
+
+[![](/wiki/shared/news/2024-12-05-osu-world-cup-2024-finals-recap/Kaiju_banner.jpg)](https://osu.ppy.sh/beatmapsets/2284698#osu/4872864)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/448/Songs/nm-y%20vs.%20Kry.exe%20-%20Kaiju.mp3">
+    </audio>
+</div>
+
+For our Tiebreaker, nm-y and [Kry.exe](https://osu.ppy.sh/beatmaps/artists/448) collaborated to make an intense and exciting electronic banger in [Kaiju](https://osu.ppy.sh/beatmapsets/2284698#osu/4872864), mapped by the trio of [Aerousea](https://osu.ppy.sh/users/15664628), [9ami](https://osu.ppy.sh/users/1499997) & [oTwinkle](https://osu.ppy.sh/users/15095654). The mechanics and tech hybrid didn't end up seeing play last weekend, but you should definitely check it out regardless!
+
+## Finals summary
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **Australia** ::{ flag=AU }:: | **7** | 2 | ::{ flag=RU }:: Russian Federation |
+| **Germany** ::{ flag=DE }:: | **7** | 3 | ::{ flag=GB }:: United Kingdom |
+| **South Korea** ::{ flag=KR }:: | **7** | 5 | ::{ flag=US }:: United States |
+| **Australia** ::{ flag=AU }:: | **7** | 3 | ::{ flag=DE }:: Germany |
+
+In the lower bracket, four teams, each with their own story to tell, sought the rarefied air of a podium spot. Australia, looking to retain their place among the top 3 — Germany, seeking a return to the glory days of 2021 and 2022 — Russia, finally having broken their so-called "semifinals curse" and making it into the top 6 for the first time since 2019 — and the United Kingdom, vying to re-establish themselves among the upper echelon of the osu!standard scene.
+
+The week opened with Australia facing off against Russia. Both teams came out firing with their first picks, but it would be Australia to muster up the first break point of the match on Russia's second pick, [Blue Night](https://osu.ppy.sh/beatmapsets/31333#osu/116006), and afterwards they never looked back. Australia dropped only one more point the rest of the way, a narrow Russian victory on [Dear The Night I Loved](https://osu.ppy.sh/beatmapsets/2284389#osu/4872029), en route to a 7–2 victory. Australia moved on to the 4th place match, while Russia was sent home.
```

Comment:
```suggestion
The week opened with Australia facing off against Russia. Both teams came out guns blazing with their first picks, but it would be Australia to muster up the first break point of the match on Russia's second pick, [Blue Night](https://osu.ppy.sh/beatmapsets/31333#osu/116006), and they would never looked back afterwards. Australia dropped only one more point the rest of the way, a narrow Russian lead on [Dear The Night I Loved](https://osu.ppy.sh/beatmapsets/2284389#osu/4872029), en route to a 7–2 victory. Australia moved on to the 4th-place match, while Russia was sent home.
```

Reply:
`and they would never look back afterwards` should be correct

Reply:
good catch